src/Compiler.php

```diff
@@ -239,13 +220,4 @@ public function getVarName(): string
     {
         return sprintf('__internal_compile_%d', $this->varNameSalt++);
     }
-
-    private function checkStringForOutput(string $string): void
```

Comment:
shouldn't we keep this deprecation ? It helps finding places where you are not yield ready. I fear that packages might mark them nodes as being yield ready without noticing they are not.

Reply:
on it